met-api/src/met_api/resources/submission.py

```diff
@@ -61,12 +61,8 @@ class Submissions(Resource):
     def post():
         """Create a new submission."""
         try:
-            user_id = TokenInfo.get_id()
             requestjson = request.get_json()
             schema = SubmissionSchema().load(requestjson)
-            schema['user_id'] = user_id
```

Comment:
I'm guessing we implement this once we have some sort of authentication for the public

Reply:
true, I don't think though the user_id will come from a jwt. I think flow is, submit email, receive survey link in email that has a valid period. Then submit survey.